app/src/main/java/ch/epfl/sdp/entity/Player.java

```diff
@@ -19,11 +19,13 @@
     public String email;
     @Exclude
     public final static double MAX_HEALTH = 100;
-    public int score;
+    public int generalScore;
+    public int currentGameScore;
     public double healthPoints;
     public double timeTraveled;
     public double distanceTraveled;
     public double speed;
+    public double distanceTraveledAtLastCheck;
```

Comment:
The problem is that we didn't calculate the distanceTraveled

Reply:
Yes, but it should not be too complicated (when we update the player's position). I thought it was a bit out of scope for the score computation.

Reply:
Yes, it should be totally doable. What we have to do is to compute the distance between the GeoPoint of the player at iteration i and the GeoPoint of the player at iteration i-1 . We can do that using the distanceTo method in the GeoPoint class. Given that the player travels on small distances, we may also not have to update this at each iteration